klog.go

```diff
@@ -874,8 +881,20 @@ func (sb *syncBuffer) Sync() error {
 	return sb.file.Sync()
 }
 
+//IsFileReachedMaxSize checks whether the file exceeds the max size limitation.
+func IsFileReachedMaxSize(bufferSize uint64) bool {
+	if logging.logFile != "" {
+		// Flag logFileMaxSize is in MB for user convenience.
+		maxSize := logging.logFileMaxSize * 1024 * 1024
+		// If logFileMaxSize is zero, we don't have limitations on the log size.
+		return maxSize > 0 && bufferSize >= maxSize
+	}
+	// If "log_file" flag is not specified, the log files will be cleaned up when reaches a fixed size.
```

Comment:
I don't understand this. If there isn't a log_file flag, then there isn't a log file, right?

Reply:
Rworded. I mean the sb.file.